apps/verticalhome/test/marionette/grid_layout_test.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+var assert = require('assert');
+
+marionette('Vertical - grid', function() {
+
+  var client = marionette.client(require(__dirname + '/client_options.js'));
+  var actions, home, system;
+
+  setup(function() {
+    actions = client.loader.getActions();
+    home = client.loader.getAppClass('verticalhome');
+    system = client.loader.getAppClass('system');
+    system.waitForStartup();
+    home.waitForLaunch();
+  });
+
+  test('apps grid is LTR in LTR language', function() {
+    var app1Location = home.getNthIcon(1).location();
+    var app2Location = home.getNthIcon(2).location();
+    var app3Location = home.getNthIcon(3).location();
+    var gridWidth = client.helper.waitForElement(home.Selectors.grid).size()
+      .width;
+
+    // check that they are all on the same line
+    assert.equal(app1Location.y, app2Location.y,
+                 'First 2 apps should be on the same line.');
+    assert.equal(app2Location.y, app3Location.y,
+                 'Last 2 apps should be on the same line.');
```

Comment:
It's nit picking, but I'd reword this not to say 'last 2', which is a little misleading - perhaps we could just assert.ok(app1Location.y == app2Location.y == app3.Location.y, 'First three apps should be on the same line')? I don't like the three-way equivalence check though either... So maybe just reword '2nd and 3rd apps should be on the same line' ? I'm fine with however you'd like to do this, including leaving as is - it's pretty clear what it's checking all the same :)